### ISSUE TITLE
Should LocalStorage Source have a remove?

### ISSUE THREAD

**Issue by CumpsD** [martyjs/marty-lib#23]
Right now `https://github.com/martyjs/marty-lib/blob/master/modules/local-storage-state-source/localStorage.js` has get and set, but the localStorage API also has `removeItem()` method.

Should the LocalStorage Source also accomodate for this?

**Comment by jhollingworth** [comment]
Yeah, it definitely should. I will try find some time to add this to the API
